src/WatchDogs.Infrastructure.FakeSource/Watcher.cs

```diff
@@ -6,8 +6,9 @@
 using System.Text;
 using System.Threading.Tasks;
 using Serilog;
+using WatchDogs.Persistence.EntityFramework;
 
-namespace WatchDogs.FakeSource;
+namespace WatchDogs.Infrastructure.FakeSource;
 
 public class Watcher : IWatcher
```

Comment:
Again, think of the assignment. We should design the application so that is has multiple data sources (and therefore Watchers). I would suggest naming this one `FakeSourceWatcher`

Reply:
easier said that done

Reply:
Done.